index.js

```diff
@@ -329,6 +329,10 @@ Connection.prototype.unref = function () {
 }
 
 Connection.prototype.address = function () {
+  if (this._socket) {
+    var addr = this._socket.address()
+    if (addr) return addr
+  }
```

Comment:
this should do in another method called remoteAddress()

Reply:
or if we wanna be like node core it needs to do:

``` js
var addr = this._socket.address()
this.remoteAddress = addr.address
this.remotePort = addr.port
```

In a constructor... (i know confusing, but that's how node core does it)

Reply:
That's fine with me, should address() be removed?